### PR TITLE
Finalized transactional emails

### PR DESCRIPTION
**Pull request by kriscooke**
Feature:
- Edits template copy for existing emails per [GGIRCS-1414](https://youtrack.button.is/issue/GGIRCS-1414)
- Adds new emails on additional interaction events
- 'CIIP' appears early in all email subject lines (above the 'fold' in email clients)
- Incremental changes added via Sqitch to email-sending DB trigger functions
  * provides data needed to construct actionable links in emails

Incidental changes:
- Adds `.gitattributes` with 'union' merge strategy for `sqitch.plan`
- Cleans up E2E tests a bit 
  * multiple eslint-disable lines in email spec --> one line at top
  * fixes/standardizes Cypress logout timing for tests logging in as specific user roles
  * adds test helper, `decoded` which _roughly_ replaces some of the email encoding characters inserted into message body text to help with asserting expected text.
- Fixes "broken" certifier link in email that was missing protocol in href [GGIRCS-1412](https://youtrack.button.is/issue/GGIRCS-1412)
- Environment variables including sender address updated to be in the long format in all environments (including the sender name part before the address)
- Adds `.gitattributes` with union auto-merge strategy for `sqitch.plan` in to help prevent conflicts now that we are doing incremental DB migrations.

To try out this branch (checking the emails):
- Run docker
- `sudo docker run -d -p 1025:1025 -p 8025:8025 mailhog/mailhog` downloads and runs MailHog
- Do some actions that generate emails
- Visit `localhost:8025` to check the MailHog inbox and see the results. **Note**: The login redirects are still broken so in order to see the correct content by following email links it is up to you to ensure you're logged in as the correct user before following a link.

### CODE REVIEW

File: schema/deploy/trigger_functions/draft_application_started.sql [hunk 1]
```diff
@@ -29,6 +29,7 @@ begin
       'sendMail',
       json_build_object(
         'type', 'draft_application_started',
+        'applicationId', new.id,
```
[the remark this answers is not in the document]
Comment (wenzowski) [reply schema/deploy/trigger_functions/draft_application_started.sql:32]: I think I saw a Teams thread today about `sqitch rework` that automates this process?

[the remark this answers is not in the document]
Comment (wenzowski) [reply schema/deploy/trigger_functions/draft_application_started.sql:32]: https://sqitch.org/docs/manual/sqitch-rework/

[the remark this answers is not in the document]
Comment (kriscooke) [reply schema/deploy/trigger_functions/draft_application_started.sql:32]: I posted that 😄 and yes, we decided to go that route

File: helm/cas-ciip-portal/values.yaml [hunk 2]
```diff
@@ -39,8 +39,8 @@ persistence:
 
 env:
   smtpConnectionString: smtp://apps.smtp.gov.bc.ca/?port=25&ignoreTLS=true&secure=false
-  senderEmail: no-reply.cas@gov.bc.ca
-  adminEmail: GHGRegulator@gov.bc.ca
+  senderEmail: BCCAS <no-reply.cas@gov.bc.ca>
```
[the remark this answers is not in the document]
Comment (kriscooke) [reply helm/cas-ciip-portal/values.yaml:46]: It looks like the decision in that card is outstanding - perhaps that can be updated once the decision or approval has been made?

**Pull request merged**